Docker/test/selenium/README.md

```diff
@@ -15,8 +15,9 @@
 - [x] Create an item with a statement and verify that querying using prefixes work correctly (wdt:P1 etc.. )
 
 ## Wikibase Client & Repo
-- [ ] Create an item on repo with sitelink to client and verify that changes are dispatched
-- [ ] Create an item on repo and verify that client can reference it using wikitext
+- [x] Create an item on repo with sitelink to client
+- [x] Create an item on repo and verify that client can reference it using wikitext
+- [ ] Verify that changes on repo are dispatched to client ( TODO figure out all use-cases )
```

Comment:
What is meant with use-cases here?

Reply:
I thought there is a bunch of cases of changes being dispatched to the client, but I could be wrong.

The stuff I've recently touched in wikibase and are aware of are ChangeModificationNotificationJob which should show up on deletion and a change of visibility.

- ChangeVisibilityNotificationJob  Job for notifying a client wiki of a batch of revision visibility changes on the repository
- ChangeDeletionNotificationJob Job for notifying a client wiki of a batch of revision deletions on the repository.

Reply:
And I'm also not entirely sure where to draw the line here, do we want to test that all of these things work or just one single thing (which might be using the same dispatch logic)? We were discussing this during the sprint planning yesterday and there is a task coming up to try to finalize this list.